src/core/Microsoft.PowerToys.Settings.UI/ViewModels/FancyZonesViewModel.cs

```diff
@@ -20,21 +20,45 @@ public class FancyZonesViewModel : Observable
 
         public ButtonClickCommand LaunchEditorEventHandler { get; set; }
 
-        public ICommand SaveColorChoiceEventHandler
+        public ICommand SaveZoneHighlightColorEventHandler
         {
             get
             {
-                return new RelayCommand<Color>(SaveColorChoice);
+                return new RelayCommand<Color>(SaveZoneHighlightColor);
+            }
+        }
+
+        public ICommand SaveBorderColorEventHandler
+        {
+            get
+            {
+                return new RelayCommand<Color>(SaveZoneBorderColor);
+            }
+        }
+
+        public ICommand SaveInActiveColorEventHandler
+        {
+            get
+            {
+                return new RelayCommand<Color>(SaveZoneInActiveColor);
             }
         }
 
         private FancyZonesSettings Settings { get; set; }
 
         public FancyZonesViewModel()
         {
-            Settings = SettingsUtils.GetSettings<FancyZonesSettings>(ModuleName);
+            try
+            {
+                Settings = SettingsUtils.GetSettings<FancyZonesSettings>(ModuleName);
```

Comment:
How about check the file exist instead of trying to read the file and gets an exception.

Reply:
but how will that help as compared to this way?

Reply:
Imo it's better to prevent an exceptions if you can. In this case suppose, GetSettings throws an `Exception`  you are assuming that the file doesn't exist and then load a default settings and save the default settings. In reality it can be any reason like some other process holding the file. And we will end up resetting the settings.

Reply:
We can discuss and look more into it as a separate PR. I don't feel that should block this.

Reply:
I see what you mean, but if something else is causing the exception then it would cause the application to crash. Now this will call for to have to try catch blocks for all possible exceptions which we should do. So the idea would be to specify in this case what kind of exception should we expect for us to create the settings file, IMO, if we fail to get the settings file it means it doesn't exist, or we failed to serialize the class model, still, this will call for us to have exceptions to cater for those cases. IMO, an if-statement is not a Safeway, we can have an exception and log telemetry for that exception instead of crashing the application.

Reply:
Hey Lavius I'm not arguing against to not catch exceptions to prevent app crashes.